ros2param/ros2param/verb/describe.py

```diff
@@ -38,16 +38,16 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             help='Consider hidden nodes as well')
         arg = parser.add_argument(
             'parameter_names', nargs='+', help='Names of the parameters')
+        parser.add_argument(
+            '--timeout', metavar='N', type=int, default=1,
```

Comment:
I propose to add a longer default timeout, e.g., 3 or 5 seconds. Nevertheless, I am uncertain if this will poolin result. In the [`spawner`](https://github.com/ros-controls/ros2_control/blob/master/controller_manager/controller_manager/spawner.py#L59-L67) of controller manager, we are waiting for 10 seconds, but check every 0.2 seconds if node is available.

Reply:
@destogl thank you for reviewing and sharing the feedback.

> I propose to add a longer default timeout, e.g., 3 or 5 seconds.

I would like to stick with `1 sec` in default, since

- user can have option to set the timeout. (this is bottom line, i think)
- how long timeout should be is really use case dependent. probably `1 sec` is too long for some users, too short for some other users...
- i think it would be probably nicer to response the result for user so that user does not wait for several seconds to see the result at 1st step.